src/Native/Runtime/MiscHelpers.cpp

```diff
@@ -114,9 +114,14 @@ COOP_PINVOKE_HELPER(HANDLE, RhGetModuleFromPointer, (PTR_VOID pPointerVal))
 
 COOP_PINVOKE_HELPER(HANDLE, RhGetModuleFromEEType, (EEType * pEEType))
 {
-#if CORERT
+#if CORERT  
     return (HANDLE)(pEEType->GetTypeManager());
 #else
+#if EETYPE_MODULE_MANAGER
```

Comment:
Why can't these 2 #if's be combined? I see that HasTypeManager is also defined for CORERT